### PR TITLE
Sara's changes, favorite food

### PR DESCRIPTION
**Pull request by sarabrandao21**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Person B should modify content here.
 
 ## Person C:
 
-Person C should modify content here.
+Favorite food.
```
Comment (ChelseaC13) [inline README.md:15]: wow

File: favorite_food.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Brazilian cheese bread are delicious.
```
Comment (ChelseaC13) [inline favorite_food.txt:1]: I see you like food

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Person B should modify content here.
 
 ## Person C:
 
-Person C should modify content here.
```
Comment (ChelseaC13) [inline README.md:15]: cool

File: favorite_food.txt (hunk 2 above)
Comment (ChelseaC13) [reply favorite_food.txt:1]: approved

**Review by ChelseaC13 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sarabrandao21) [reply README.md:15]: Hello Chelsea

**Pull request merged**